### PR TITLE
update(purge-tool): Add warn document not deleted

### PR DESCRIPTION
**Pull request by marielle-spiteri**
* Document not deleted warning

### CODE REVIEW

File: modules/ROOT/pages/purge-tool.adoc [hunk 1]
```diff
@@ -13,6 +13,12 @@ For Enterprise, Performance, Efficiency, and Teamwork editions only. +
 This tool is not available publicly. Use https://customer.bonitasoft.com/download/request[Bonita Customer Portal] to download it.
 ====
 
+[WARNING]
+====
+
```
Comment (alachambre) [inline modules/ROOT/pages/purge-tool.adoc:18]:
```suggestion
```

**Comment by alachambre** [comment]
Can an engine guy (or their supreme leader @pabloalonso) have a look? I don't know enough the purge tool to merge this 2 weeks pull request by myself :)

**Comment by marielle-spiteri** [comment]
Hi @alachambre, I can guarantee you that this warning is accurate, you can trust me, here's the proof: RUNTIME-147. 
I know it's a surprising behavior, to say the least, but it's 100% true and I think our subscription users should be aware of that when reading the documentation ;) 
I will close this PR because I created a new one that is more complete: #1545. Could you please merge that one instead?

**Comment by alachambre** [comment]
I just suggested to remove a blank line, not the warning don't worry :) 

During the migration from markdown to asciidoc, blank lines where added for no reason in the admonition blocks and every one is copy pasting the blank line now :'(

**Pull request closed without merging**